os2mo_data_import/os2mo_helpers/mora_helpers.py

```diff
@@ -509,11 +515,12 @@ def read_ou_manager(self, unit_uuid, inherit=False):
         return manager
 
     def read_organisation_people(self, org_uuid, person_type='engagement',
-                                 split_name=True, read_all=False, skip_past=False):
+                                 split_name=True, read_all=False, skip_past=False, primary_types=[]):
         """ Read all employees in an ou. If the same employee is listed
         more than once, only the latest listing will be included.
         :param org_uuid: UUID of the OU to find emplyees in.
         :read_all: Read all engagements, not only the present ones.
+        :primary_types: list of primary_type UUIDs allowed for export
         :return: The list of emplyoees
         """
```

Comment:
Det er et anti-pattern at have default values som `[]`.
```suggestion
                                 split_name=True, read_all=False, skip_past=False, primary_types=None):
        """ Read all employees in an ou. If the same employee is listed
        more than once, only the latest listing will be included.
        :param org_uuid: UUID of the OU to find emplyees in.
        :read_all: Read all engagements, not only the present ones.
        :primary_types: list of primary_type UUIDs allowed for export
        :return: The list of emplyoees
        """
        primary_types = primary_types or []
```
Grunden herfor er lidt overraskende, men kan ses med følgende kode stump:
```
>>> def func(listy=[]):
...     listy.append('a')
...     return listy
... 
>>> print(func([1]))
[1, 'a']
>>> print(func([1]))
[1, 'a']
```
Hvor koden virker som forventet, men med følgende:
```
>>> print(func())
['a']
>>> print(func())
['a', 'a']
>>> print(func())
['a', 'a', 'a']
```
Virker koden aldeles uventet, fordi man har side-effekter på default argumentet.